bin/proving-service/src/commands/worker.rs

```diff
@@ -92,15 +93,15 @@ impl StartWorker {
         );
 
         // Create a health reporter
-        let (mut health_reporter, health_service) = health_reporter();
+        let (health_reporter, health_service) = health_reporter();
 
         // Mark the service as serving
         health_reporter.set_serving::<ApiServer<RpcListener>>().await;
 
         tonic::transport::Server::builder()
             .accept_http1(true)
-            .add_service(tonic_web::enable(rpc.api_service))
-            .add_service(tonic_web::enable(rpc.status_service))
+            .add_service(tonic_web::GrpcWebLayer::new().layer(rpc.api_service))
+            .add_service(tonic_web::GrpcWebLayer::new().layer(rpc.status_service))
```

Comment:
I am not sure if it works but according to tonic docs this should be done with:
```suggestion
            .layer(GrpcWebLayer::new())
            .add_service(rpc.api_service)
            .add_service(rpc.status_service)
```

Reply:
Oh, my way was working but if this works it would be much better. Will test it and come back with the results.

Reply:
I think we still need to add the `.layer(GrpcWebLayer::new())` line